### PR TITLE
docs(release-policy): split release policy

### PR DESCRIPTION
**Pull request by akeller**
Updating release policy page to point to new https://camunda.com/release-policy/

### CODE REVIEW

File: docs/reference/release-policy.md [hunk 1]
```diff
@@ -28,18 +22,18 @@ A cluster usually consists of:
...
+For components **outside of the cluster**, we release new versions continuously and update customers to the latest version automatically whenever it is ready to be shipped.
 
 These components include:
 
 * Modeler (Web)
 * Connectors
 * Console
 
-For components inside a cluster, Camunda provides two channels for provisioning and follows above release policy:
+For components inside a **cluster**, Camunda provides two channels for provisioning and follows above release policy:
```
Comment (christinaausley) [inline docs/reference/release-policy.md:33]: What do we mean by "follows above release policy" here? Do we mean follows **the** above release policy?

**Comment by christinaausley** [comment]
@akeller One minor adjustment in my latest commit, and one lingering comment. Will approve once resolved as there are no other adjustments on my end.

**Comment by akeller** [comment]
> and one lingering comment

@christinaausley I'm not seeing a comment, what am I missing?

**Comment by christinaausley** [comment]
> > and one lingering comment
> 
> @christinaausley I'm not seeing a comment, what am I missing?

Oops, comment was still in `Pending` as I added it to an official review.

**Comment by akeller** [comment]
Let's hold off on merging until @felix-mueller takes a look just to confirm the info is correct.

**Comment by akeller** [comment]
@felix-mueller can you review this before we merge? I've seen a lot of questions about release policy recently and I want to make sure this is correct before we push it.

**Review by felix-mueller — APPROVED** [review]

**Pull request merged**